cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -1082,6 +1082,15 @@ rules:
   - apiGroups: [""]
     resources: ["configmaps"]
     verbs: ["get"]
+  - apiGroups: ["replication.storage.openshift.io"]
+    resources: ["volumereplications"]
+    verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]
+  - apiGroups: ["replication.storage.openshift.io"]
```

Comment:
Would replication only work on openshift? Or is the RBAC just defined in this namespace at it will still work on K8s?

Reply:
This is just an API group it should work on kubernetes as well.